samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_app_location.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <drivers/gps.h>
+#include <net/lwm2m.h>
+#include <net/lwm2m_path.h>
+
+#include "app_gps_event.h"
+#include "lwm2m_app_utils.h"
+
+#define MODULE lwm2m_app_loc
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_APP_LOG_LEVEL);
```

Comment:
Just give the name of the module directly here.

Reply:
The MODULE macro is also used at the end of the file with the EVENT_LISTENER() and EVENT_SUBSCRIBE() macros from Event Manager. We could just write the name directly, but I think this is what is done in Asset Tracker v2

Reply:
Ok, sounds like a good reason.